arm9/asm/scrcmd_18.s

```diff
@@ -615,8 +618,8 @@ _020447F2:
 	pop {r4-r6, pc}
 	.balign 4
 
-	thumb_func_start FUN_02044800
-FUN_02044800: ; 0x02044800
+	thumb_func_start ScrCmd_findpartymonwithnature
```

Comment:
possibly hard to distinguish words here?

Reply:
crameverythingtogetherlowercase is the standard for script commands

Reply:
fair enough, I can just see it being a bit difficult to separate words